packages/datatrak-web/src/components/Tile.tsx

```diff
@@ -57,19 +57,26 @@ const Text = styled(Typography)`
 interface TileProps {
   title: string;
   text: string;
-  to: string;
+  to?: string;
   tooltip?: ReactNode;
-  children: ReactNode;
+  children?: ReactNode;
+  Icon?: ComponentType;
+  onClick?: () => void;
 }
 
-export const Tile = ({ title, text, children, to, tooltip }: TileProps) => {
+export const Tile = ({ title, text, children, to, tooltip, Icon, onClick }: TileProps) => {
   return (
-    <ButtonWrapper component={RouterLink} to={to} tooltip={tooltip}>
-      <SurveyTickIcon />
+    <ButtonWrapper
+      component={to ? RouterLink : undefined}
```

Comment:
```suggestion

```
I don't think that we need this logic here since it's happening in the button component right?